contrib/create-waku/src/cli.ts

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env node
+
+import fs from 'node:fs'
+import path from 'node:path'
+import prompts from 'prompts'
+import { red, green, bold } from 'kolorist'
+
+import { emptyDir } from './emptyDir'
```

Comment:
We need an extension for module=nodenext. `.js` not `.ts`.
```suggestion
import { emptyDir } from './emptyDir.js'
```

Reply:
Do you mean adding ``.js`` to all importing files?

Reply:
https://www.typescriptlang.org/docs/handbook/esm-node.html

> As a result, it will have to be rewritten to use the extension of the output of foo.ts - so bar.ts will instead have to import from ./foo.js.